### ISSUE TITLE
snapshot: warning: unknown warning option '-Wno-clobbered';

### ISSUE THREAD

**Issue by alfredh** [baresip/baresip#1352]
new compiler warning on OSX 10.14:

```
  CC [M]  build-x86_64/modules/snapshot/snapshot.o
warning: unknown warning option '-Wno-clobbered'; did you mean '-Wno-consumed'? [-Wunknown-warning-option]
1 warning generated.
  CC [M]  build-x86_64/modules/snapshot/png_vf.o
warning: unknown warning option '-Wno-clobbered'; did you mean '-Wno-consumed'? [-Wunknown-warning-option]
1 warning generated.
  LD [M]  snapshot.so
```

**Comment by sreimers** [comment]
Looks like the gcc ifeq check fails:

https://github.com/baresip/baresip/blob/master/modules/snapshot/module.mk#L9
```mk
ifeq ($(findstring gcc, $(CC)), gcc)
$(MOD)_CFLAGS	+= -Wno-clobbered
endif
```

does this help?

```mk
ifneq (,$(findstring gcc, $(CC)))
$(MOD)_CFLAGS	+= -Wno-clobbered
endif
```

**Comment by alfredh** [comment]
fixed in 3e6620e4363796704a84b7762b071ba591e12cfb

thanks @sreimers !
